### PR TITLE
Cassidy

### PR DESCRIPTION
**Pull request by cassidycr94**
Added a few new commands and tested them in the test server.

### CODE REVIEW

**Comment by Kiri23** [comment]
Can you fix this merge conflict you need to download the latest project from github and add the new response that you made

**Comment by cassidycr94** [comment]
Looks Good!

File: bot.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ if ((input.includes("HI BOT") )) {
 
 
 //Login to Discord using oauth
-bot.login('TOKEN_KEY');
+bot.login('MjQ1MzkwMDg0NDgyOTI0NTQ2.CwQefg.p2rkiB8vIb5WHjbCyfCE3K1DA4s');
```
Comment (Kiri23) [inline bot.js]: For security purpose you can hide this(token key)

File: bot.js [hunk 2]
```diff
@@ -30,86 +30,90 @@ request(StackOverflowSearchUrl, function (error, response, body) {
...
 
-// TODO Condition = What the bot can do. Create a Response for this
+//TODO Create a void method to separate the code from here
 
 //TODO Condition = I can get all the course that I paid in Udmey within the Desvlope app. Proccess = str.includes("COURSE") && str.includes("UDEMY") && str.includes("DEVSLOPE APP"). Response = No --Custom Response--
 
 //TODO Make a Grahical User Interface to add condition and Response an validate the input instead of adding each condition and Response in code.
 
-//TODO condition = when the --mac app-- --devslope mac app-- will be ready. response = the mac app will be ready in december 31
+
```
Comment (Kiri23) [inline bot.js]: 1. Missing response that display all available commands 
2.  Missing response that list all the course from devslopes 
3.  Missing response that display coupon code 
4. Missing safety check 

#17 here it is the pull request that have the responses can you please include it.

Comment (cassidycr94) [inline bot.js]: Oh I see yea I will pull that and re submit it tonight

Comment (Kiri23) [inline bot.js]: 😎️👍️

**Comment by Kiri23** [comment]
done

**Pull request merged**